src/core/server/base.js

```diff
@@ -19,20 +20,37 @@ import config from 'config';
 import { setJWT } from 'core/actions';
 
 const env = config.util.getEnv('NODE_ENV');
-const isDeployed = ['stage', 'dev', 'production'].indexOf(env) > -1;
+const isDeployed = config.get('isDeployed');
+const isDevelopment = config.get('isDevelopment');
 
 const errorString = 'Internal Server Error';
 const appName = config.get('appName');
 
+const log = bunyan.createLogger({
+  name: 'server',
+  app: appName,
+  serializers: bunyan.stdSerializers,
+});
+
 // Globals (these are set by definePlugin for client-side builds).
 global.CLIENT = false;
 global.SERVER = true;
-global.DEVELOPMENT = env === 'development';
+global.DEVELOPMENT = isDevelopment;
```

Comment:
I think we can kill these and the definePlugin versions of the same in the webpack conf since the config exposes what we need. Also I'm not sure if we need the CLIENT and SERVER ones? Maybe those are still useful?